### DISCUSSION TITLE
2023 School on the SSCHA code!

### DISCUSSION THREAD

**Discussion by mesonepigreco** [SSCHAcode/python-sscha#124] [General]
I am excited to announce that we are organizing a one-week school on the sscha code in San Sebastian, Spain, from June 26-30 2023
The school will provide hands-on sessions focused on the computation of several critical aspects of material science and condensed matter physics, including:

- Phase diagrams (first and second-order phase transitions; Ferroelectric and Peierls transitions).

- Thermodynamic properties at finite temperature (Entropy, thermal expansion, bulk modulus).

- Vibrational spectroscopy (Raman & IR).

- Thermal transport in the overdamped regime.

- Electron-phonon coupling and superconductivity.

Attending the school will help enrich your toolbox, especially since accounting for anharmonicity in many materials is becoming the new state-of-art is "expected" by papers published in high-profile journals.

Participants can present their research through a poster or an oral presentation. The registration fee for the school is only 150€ and includes the cost of the social dinner, coffee breaks, and free buffet during the poster session.

San Sebastian is a stunning location, particularly in the summer, with the sea and beautiful beaches.
For more information about the program, accommodation, and registration, please visit the following link: http://sscha.eu/Schools/2023/home/

I encourage you to join us for this excellent opportunity to learn, network, and enjoy the beauty of San Sebastian.

**Comment by jhung12** [comment]
Will the lecture be recorded?

**Reply by mesonepigreco** [reply]
Most likely, yes, but we have to coordinate with the local infrastructure to determine the best way. If you have the possibility, we strongly suggest attending in person :)

**Reply by mesonepigreco** [reply]
Lectures and hands-on sessions have been published on Youtube! 
https://www.youtube.com/@SSCHAcode/playlists

Subscribe to the SSCHAcode channel for updates :smile:

**Reply by jhung12** [reply]
Thanks!!
